### ISSUE TITLE
Hebrew calendar too?

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [aminography/PrimeDatePicker#1]
I've noticed the screenshots show civil and Persian calendars.
Is it also possible to use Hebrew calendar (AKA Jewish calendar) ?

**Comment by aminography** [comment]
Yes, it is possible to add Hebrew calendar too, I'll consider it for future development.

**Comment by AndroidDeveloperLB** [comment]
Thank you.
Note that just like Arabic and Persian, Hebrew is an RTL language (right to left).
And, just like every calendar, it has its own unique and weird rules.
Saturday is the end of the week, and it's when people don't work (sometimes also Friday), as opposed to the civil calendar that Saturday and Sunday are when people don't work

**Comment by AndroidDeveloperLB** [comment]
Why close this? Was it added? Was it decided not to add? What happened?

**Comment by aminography** [comment]
No, No, No. Nothing happened dude;

**Comment by AndroidDeveloperLB** [comment]
Oh ok.
